packages/cyberstorm/src/components/TextInput/TextInput.tsx

```diff
@@ -4,14 +4,19 @@ import styles from "./TextInput.module.css";
 import { Icon } from "../Icon/Icon";
 import { classnames } from "../../utils/utils";
 
-export interface TextInputProps {
+export interface TextInputProps
+  extends React.ComponentPropsWithRef<"input">,
+    React.PropsWithChildren {
   placeHolder?: string;
   leftIcon?: JSX.Element;
   id?: string;
```

Comment:
Now that we use `ComponentPropsWithRef<"input">`, doesn't that mean we don't need to manually define id, type, name, onBlur, or onChange, unless we want to override the types of whatever values `<input>` accepts normally? (And placeHolder, if we rename it to placeholder.)